source/core/Date.cpp

```diff
@@ -309,30 +305,23 @@ namespace Vireo {
         }
     }
 
+
     //------------------------------------------------------------
-    Int32 Date::getLocaletimeZone()
+    Int32 Date::getLocaletimeZone(Int64 utcTime)
     {
-#if kVireoOS_emscripten
-        TempStackCString result;
-        result.AppendCStr(jsTimestampGetTimeZoneOffset());
-        EventLog log(EventLog::DevNull);
-        SubString valueString(result.Begin(), result.End());
-        TDViaParser parser(THREAD_TADM(), &valueString, &log, 1);
-        TypeRef parseType = THREAD_TADM()->FindType(tsInt32Type);
-        Int32 minutes;
-        parser.ParseData(parseType, &minutes);
-        _systemLocaleTimeZone = -(minutes * kSecondsPerMinute);
-        // flipping the sign of the time zone since JS runtime will be positive
-        // for being behind UTC and negative for being ahead. ctime assumes
-        // negative for behind and postive for ahead. We attempt to match the ctime result.
-        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset
-#elif (kVireoOS_linuxU || kVireoOS_macosxU)
+// #if kVireoOS_emscripten formerly here which was using jsTimestampGetTimeZoneOffset has been deleted; the localtime_r emulation works correctly
```

Comment:
I see both of your points. It is not a bad idea to put the current date against comments like this. After several months or a year of testing and shipping products and  having good runtime, it might be ok to delete the comment then.